src/components/Error/ErrorScreen.js

```diff
@@ -23,23 +24,17 @@ function ErrorScreen({ children }) {
           url(${eagleSvg});
       `}
     >
-      <ButtonBase
-        href="/"
-        external={false}
+      <HomeButton
+        onClick={() => {
+          // force a refresh rather than only changing the hash
+          window.location = '/'
```

Comment:
This seems a bit brittle; maybe we can include this as a utility in the router?

Reply:
We could yes, temporarily… but we have many `window.location` in the codebase: the plan is really to refactor the router and replace all the `window.location` references by a generic `()` / `const { go, route } = useRoute()` (or something similar) coming from the router.

Reply:
That sounds good; let's do this with #927!